iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/string.inl

```diff
@@ -535,57 +538,60 @@ string<Capacity>::insert(const uint64_t pos, const T& str, const uint64_t count)
     return true;
 }
 
+// AXIVION Next Construct AutosarC++19_03-A14.7.1:
```

Comment:
I couldn't find a violation of this rule. Can you please check if this suppression can be removed?

Reply:
@FerdinandSpitzschnueffler right, it is suppressed globally